Translators/WZDx/main.py

```diff
@@ -82,3 +85,45 @@ def translateWzdxTIM(request):
 
     tims = translate(request.get_json())
     return (json.dumps(tims), 200, headers)
+
+def entry(request):
+    logging.info('TIM Translator Timer Called...')
+    if request.method == 'OPTIONS':
+        headers = {
+            'Access-Control-Allow-Origin': '*',
+            'Access-Control-Allow-Methods': 'POST',
+            'Access-Control-Allow-Headers': 'Content-Type',
+            'Access-Control-Max-Age': '3600'
+        }
+
+        return ('', 204, headers)
+
+    headers = {
+        'Access-Control-Allow-Origin': '*'
+    }
+
+    # Scrape the CDOT endpoint to get current list of WZDX features
+    geoJSON =  json.loads(requests.get(f'https://{os.getenv("WZDX_ENDPOINT")}/api/v1/wzdx?apiKey={os.getenv("WZDX_API_KEY")}').content.decode('utf-8'))
+    
+    tim_list = translate(geoJSON)
+
+    logging.info('Pushing TIMs to ODE...')
+
+    errNo = 0
+    for tim in tim_list:
+        for msg in tim["tim"]["dataframes"]:

```

Comment:
Would it be better to handle this workZone and milliseconds issues in the translate methods instead?

Reply:
Moved these checks to the translate method